app/assets/stylesheets/govuk_publishing_components/components/_checkboxes.scss

```diff
@@ -1,2 +1,16 @@
 @import "helpers/govuk-frontend-settings";
 @import "govuk-frontend/components/checkboxes/checkboxes";
+
+.govuk-checkboxes--nested {
+  margin-left: (govuk-spacing(3) + 3px);
+  padding-left: govuk-spacing(4);
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
```

Comment:
Don't think these browser-prefix versions are needed any more. Firefox is evergreen and webkit browsers have supported this for ages.